packages/notifi-react-card/lib/components/subscription/EventTypeHealthCheckRow.tsx

```diff
@@ -108,19 +145,21 @@ export const EventTypeHealthCheckRow: React.FC<
     if (loading) {
       return;
     }
+    setErrorMessage('');
     e.target.placeholder = 'Custom';
     const ratioNumber = getParsedInputNumber(e.target.value);
-    if (ratioNumber && ratioNumber >= 1 && ratioNumber <= 99) {
-      // TODO: hook up API call here
-      // instantSubscribe({
-      //   alertConfiguration: healthThresholdConfiguration({
-      //     alertFrequency: 'ALWAYS',
-      //     percentage: ratioNumber,
-      //   }),
-      //   alertName: alertName,
-      // });
+    if (ratioNumber && ratioNumber >= 1 && ratioNumber <= 99 && customValue) {
+      instantSubscribe({
+        alertConfiguration: healthThresholdConfiguration({
+          alertFrequency: 'DAILY',
```

Comment:
now that i think about it, it would be nice if we can get the alertFrequency from the card config. Let's do that as a followup